python/commons/lab.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import os
+import numpy as np
+
+import datajoint as dj
+
+schema = dj.schema('common_lab', locals())
+
+@schema
+class Paths(dj.Lookup):
+    definition = ...
+
+    def getLocalPath(self, path, local_os=None):
```

Comment:
Complete oversight on my side - it does indeed use `self` as this is acting as placeholder class for DataJoint table implemented in Matlab.